content/docs/6.1/develop/clients/go.md

```diff
@@ -8,8 +8,237 @@ menu:
+
+To actually commit the transaction to TiKV, you need to call `Commit` to trigger the commit process.
+
+If the transaction does not need to commit, for optimistic transactions, you can just discard the transaction instance, for pessimistic transactions you need to actively call the `Rollback()` method to clean up the data previously sent to TiKV.
+
+```go
+if err := txn.Commit(context.TODO()); err != nil {
+    // ... handle error ...
+}
+// ... commit success ...
+```
+
+### Snapshots (Read-Only Transactions)
+
+If you want to create a read-only transaction, you can use `GetSnapshot` method to create a snapshot. A `Snapshot` is more lightweight than a transaction.
+
+```go
+ts, err := client.CurrentTimestamp("global")
+if err != nil {
+    // ... handle error ...
+}
+snapshot, err := client.GetSnapshot(ts)
+if err != nil {
+    // ... handle error ...
+}
+v, err := snapshot.Get(context.TODO(), []byte("foo"))
+// ... handle Get result ...
+```
+
+Snapshot can also be extracted from a existed transaction.
+
+```go
+snapshot := txn.GetSnapshot()
+// ... use snapshot ...
+```
+
+
+## Try the Raw key-value API
+
+### Create client
+
+After starting a TiKV cluster successfully, we can use PD's address list to create a client to interact with it.
+
+```go
+import (
+    "github.com/tikv/client-go/v2/config"
+    "github.com/tikv/client-go/v2/rawkv"
+)
+
+client, err := rawkv.NewClient(context.TODO(), []string{"127.0.0.1:2379"}, config.Security{})
+if err != nil {
+    // ... handle error ...
+}
+```
+
+### Closing Client
+
+When you are done with a client, you need to gracefully close the client to finish pending tasks and terminate all background jobs.
+
+```go
+if err := client.Close(); err != nil {
+    // ... handle error ...
+}
+```
+
+### Single Key Read/Write
+
+`RawKV` provides `Get`, `Put` and `Delete` methods to read and write a single key.
+
+```go
+v, err := client.Get(context.TODO(), []byte("key"))
+if err != nil {
+    // ... handle error ...
+}
+// ... handle value v ...
+
+err = client.Put(context.TODO(), []byte("key"), []byte("value"))
+if err != nil {
+    // ... handle error ...
+}
+
+err = client.Delete(context.TODO(), []byte("key"))
+if err != nil {
+    // ... handle error ...
+}
+```
+
+### Iterations
+Like `txnkv`, there are also `Scan` and `ReverseScan` methods to iterate over a range of keys.
+
+```go
+keys, values, err := client.Scan(context.TODO(), []byte("begin"), []byte("end"), 10)
+if err != nil {
+    // ... handle error ...
+}
+// ... handle keys, values ...
+
+keys, values, err := client.ReverseScan(context.TODO(), []byte("begin"), []byte("end"), 10)
+if err != nil {
+    // ... handle error ...
+}
+// ... handle keys, values ...
+```
+
+### Batch Operations
+
+`RawKV` also supports batch operations using batch. Note that since `RawKV` is not transaction guaranteed, we do not guarantee that all writes will succeed or fail at the same time when these keys are distributed across multiple regions.
```

Comment:
```suggestion
`RawKV` also supports batch operations using batch. Note that since `RawKV` does not provide transaction semantic, we do not guarantee that all writes will succeed or fail at the same time when these keys are distributed across multiple regions.
```